scripts/digest_inputs

```diff
@@ -145,7 +159,7 @@ add_vars PKG_NAME PKG_NAME_GUESSED
 # always be within the filesystem tree rooted at workdir
 # because we can't access files outside of this tree.
 # Hence this is prefixed with the workdir supplied.
-ZIP_LOCATION="$WORKDIR/dist/$OS/$ARCH"
+ZIP_LOCATION="$WORKDIR/dist/$OS_VERSION/$ARCH"
```

Comment:
This is the default path we look on disk for product's builds right?

Are we expecting product teams (based on L143) to create their build outputs in windows specific paths? I was thinknig the os_version details are a docker issue and they wouldn't care/need to know.

I'd be worried this would break the default parameters if we did this.

Reply:
Yes and that's the path where we download the binary artifacts to. We set the `TARGETOS` in the windows build argument to `OS_VERSION` [here](https://github.com/hashicorp/actions-docker-build/blob/releng/windows-test/scripts/docker_build#L62). They need to match, the `COPY` layer would fail since it wouldn't find the binary correctly. Unless we change the docker build script to set `TARGETOS` to just `OS`. Either way, product team wouldn't need to know since they're all being calculated under the hood.